packages/fx-core/src/core/middleware/utils/debug/taskMigrator.ts

```diff
@@ -92,17 +99,18 @@ export async function migrateTransparentLocalTunnel(context: DebugMigrationConte
     }
 
     if (isCommentObject(task["args"])) {
-      const comment = `
-        {
-          // Keep consistency with upgraded configuration.
-        }
-      `;
-      task["args"]["type"] = TunnelType.ngrok;
-      task["args"]["env"] = "local";
-      task["args"]["writeToEnvironmentFile"] = assign(parse(comment), {
-        endpoint: context.placeholderMapping.botEndpoint,
-        domain: context.placeholderMapping.botDomain,
-      });
+      if (task["args"]["ngrokArgs"] === TaskDefaultValue.startLocalTunnel.ngrokArgs) {
```

Comment:
Will add port-only customization to this case? E.g. `http 3333 ...` arg?

Reply:
> Will add port-only customization to this case? E.g. `http 3333 ...` arg?

Yes, I will update the logic here.